src/global-styles.scss

```diff
@@ -1,14 +1,71 @@
-$example-color: cornflowerblue;
+// gray-700
+// $bg-color: #374151;
 
-.example-class {
-    color: $example-color;
+// zinc-900
+$bg-color: #18181b;
+$text-color: #f8fafc;
+
+// gray-800
+// $card-lv-1-color: #1f2937;
+
+// zinc-700
+$card-lv-1-color: #3f3f46;
+
+
+
+html {
+    background-color: $bg-color;
+    color: $text-color;
+    height: 100%;
```

Comment:
height is doing nothing

Reply:
uncommented:

![image](https://github.com/user-attachments/assets/10ec889e-3b80-4373-a1f9-fe9d4ee49f4a)

height commented out:

![image](https://github.com/user-attachments/assets/2e8f72dd-5b53-4861-9f71-58fe9b3fc3d3)